lib/database.rb

```diff
@@ -92,10 +93,13 @@ def get_data(type, value)
     nil
   end
 
-  def get_id(type, value)
-    data = get_data(type, value)
-    if data
-      return data["id"]
+  # SiteSetting を使って toc url を正規化してマッチングする。
+  # get_data("toc_url", url) だと、アドレスが仕様変更した場合に、
+  # 古いままのデータとマッチングしなくなるため
+  def get_data_by_toc_url(toc_url, site_setting)
+    @database.each do |_, data|
```

Comment:
__[RuboCop]__

Performance/HashEachMethods: Use `each_value` instead of `each`.


<!-- Comment by [SideCI](https://sideci.com) -->